### PR TITLE
generate Integration tests log in the right output folder

### PR DESCRIPTION
**Pull request by carloscaldas**
Issue: https://github.com/LBNL-UCB-STI/beam/issues/2415

---
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/2420)

### CODE REVIEW

File: src/main/java/beam/utils/LoggingUtil.java [hunk 1]
```diff
@@ -30,24 +34,23 @@ public static void initLogger(String outputDirectory, boolean keepConsoleAppende
             }
             // ################################################
 
-            InputStream resourceAsStream = context.getClass().getClassLoader().getResourceAsStream(logFileName);
+            InputStream resourceAsStream = context.getClass().getClassLoader().getResourceAsStream(logbackConfigFile);
             if (resourceAsStream != null) {
                 try {
+                    context.reset();
```
Comment (REASY) [inline src/main/java/beam/utils/LoggingUtil.java]: Why did you move the these lines upper? According the documentation http://logback.qos.ch/faq.html#sharedConfiguration
```java
LoggerContext context = (LoggerContext) LoggerFactory.getILoggerFactory();
  JoranConfigurator jc = new JoranConfigurator();
  jc.setContext(context);
  context.reset(); // override default configuration
  // inject the name of the current application as "application-name"
  // property of the LoggerContext
  context.putProperty("application-name", NAME_OF_CURRENT_APPLICATION);
  jc.doConfigure("/path/to/the/above/configuration/file.xml"); 
```

File: src/test/resources/logback-test.xml [hunk 2]
```diff
@@ -15,8 +15,18 @@
...
+    <appender name="FILE" class="ch.qos.logback.core.FileAppender">
+        <File>${log-path}/beamLog.out</File>
+        <layout class="ch.qos.logback.classic.PatternLayout">
+            <Pattern>
+                %d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n
+            </Pattern>
+        </layout>
+    </appender>
+
+    <root level="INFO">
```
Comment (wrashid) [inline src/test/resources/logback-test.xml]: beamLog.out should be info, but console should be Error level

File: src/main/java/beam/utils/LoggingUtil.java (hunk 1 above)
Comment (carloscaldas) [reply src/main/java/beam/utils/LoggingUtil.java]: just code organization. Does not makes difference since `jc.setContext(context);` only saves the reference
```Java
    public void setContext(Context context) {
        if (this.context == null) {
            this.context = context;
        } else if (this.context != context) {
            throw new IllegalStateException("Context has been already set");
        }
    }
```

File: src/test/resources/logback-test.xml (hunk 2 above)
Comment (carloscaldas) [reply src/test/resources/logback-test.xml]: done
